packages/cli/cli/versions.yml

```diff
@@ -1,3 +1,11 @@
+- changelogEntry:
+  - summary: |
+      The CLI now supports checking for broken links in your docs. You will see warnings in `fern check` if your docs link to any page that can't be resolved, and
+      you can also run `fern docs broken-links` to only check for broken links.
+    type: feat
```

Comment:
@rpc333 no pressure -- but would be great to document all the different CLI commands you showed or link to a page in the documentation for this

Reply:
@dsinghvi I added some extra info to changelog + updated the documentation, but the latter in particular might want some extra attention as a follow-up based on conversation @abvthecity @amckinney and I were having in Slack